### PR TITLE
feat(backend): Add orderBy to GetOrganizationListParams

### PR DESCRIPTION
**Pull request by IGassmann**
## Description

This adds the `orderBy` parameter to `GetOrganizationListParams`.

The `getOrganizationList` method in the `OrganizationApi` was missing the `orderBy` parameter, as @mario-jerkovic correctly pointed out in https://github.com/orgs/clerk/discussions/3152.

The related documentation PR: https://github.com/clerk/clerk-docs/pull/891

## Checklist

- [x] `npm test` runs as expected.
- [x] `npm run build` runs as expected.
- [x] (If applicable) [JSDoc comments](https://jsdoc.app/about-getting-started.html) have been added or updated for any package exports
- [x] (If applicable) [Documentation](https://github.com/clerk/clerk-docs) has been updated

## Type of change

- [ ] 🐛 Bug fix
- [x] 🌟 New feature
- [ ] 🔨 Breaking change
- [ ] 📖 Refactoring / dependency upgrade / documentation
- [ ] other:

### CODE REVIEW

File: packages/backend/src/api/endpoints/OrganizationApi.ts [hunk 1]
```diff
@@ -22,6 +22,16 @@ type MetadataParams<TPublic = OrganizationPublicMetadata, TPrivate = Organizatio
...
+  orderBy?:
+    | 'name'
+    | '+name'
+    | '-name'
+    | 'created_at'
+    | '+created_at'
+    | '-created_at'
+    | 'members_count'
+    | '+members_count'
+    | '-members_count';
```
Comment (IGassmann) [inline packages/backend/src/api/endpoints/OrganizationApi.ts:34]: The available values corresponds to the [ones documented in the Backend API Reference](https://clerk.com/docs/reference/backend-api/tag/Organizations#operation/ListOrganizations!in=query&path=order_by&t=request).

**Review by dimkl — APPROVED** [review]
Nice! 🚀 Thank for the contribution.
I will open an internal ticket so we can also add/update the `orderBy` parameter in the following endpoints but feel free to open a PR for those 2 and i will review it:
- https://clerk.com/docs/reference/backend-api/tag/Organization-Memberships#operation/ListOrganizationMemberships!in=query&path=order_by&t=request
- https://clerk.com/docs/reference/backend-api/tag/Users#operation/GetUserList!in=query&path=order_by&t=request

PS: this change will be added to the latest beta version. In order to backport this to the current latest version you need to open a PR with `release/v4` as base branch but since we will release the beta version as stable soon, i would consider using the latest `beta` instead. For beta docs visit: https://beta.clerk.com/docs

**Comment by IGassmann** [comment]
@dimkl it looks like you have to approve the running of the GitHub workflows, before I'm able to merge.

**Review by nikosdouvlis — APPROVED** [review]
👏🏻

**Comment by dimkl** [comment]
@IGassmann even if i approve it, the actions won't run. Nikos will merge it :)

**Comment by nikosdouvlis** [comment]
> @dimkl it looks like you have to approve the running of the GitHub workflows, before I'm able to merge.

We have a few GH variables that cannot be shared with forked repos - we will most liekly push the fix this week, so I will manually merge and run CICD for now :) 

Thank you for your contribution!

**Pull request merged**